hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerSet.java

```diff
@@ -65,10 +69,19 @@ public class ContainerSet implements Iterable<Container<?>> {
       new ConcurrentSkipListMap<>();
   private Clock clock;
   private long recoveringTimeout;
+  private final Table<Long, State> containerIdsTable;
+
+  public ContainerSet(Table<Long, State> continerIdsTable, long recoveringTimeout) {
```

Comment:
If we kept the existing constructor and marked it a visibleForTest, then make it create a "test version" of this class, would it reduce all the test files that need to be changed?

```
public ContainerSet(long recoveringTimeout) {
    this(ImMemoryTable, recoveringTimeout, true / false);
}
```

Reply:
done